### ISSUE TITLE
How to distribute the examples

### ISSUE THREAD

**Issue by jklymak** [c-proof/pyglider#72]
python packages can have data, but usually that is for things the routines need at runtime.  Not 100% sure how we should distribute the examples here.  Maybe as a tarball in the `doc/_static/`?  Not sure what other libraries do for examples like this that require significant data.

**Comment by jklymak** [comment]
Ok that's a little different because they just need a file at a time and they want the datasets to run in an example. 

We have to serve a whole directory, and I'm not sure we need to show much after the netcdf files are created.  At that point they are just netcdf files.  

Keeping the test data in a separate repo isn't a bad idea and maybe is all we need.  Users and the tests can just install that repo locally with git clone.  My only concern with that is some folks won't be thrilled with git clone and we prob also have to serve a tarball somewhere.

**Comment by callumrollo** [comment]
I think a separate repo would be a good idea. Should we just make a new repo with the current example directories? We could easily tarball the whole thing and make it available with a link in the README of pyglider

**Comment by jklymak** [comment]
Yeah, I think that's easiest. I'm a little ignorant how to get the tests to download that directory, but I think I can figure it out.

**Comment by jklymak** [comment]
OK, repo here... https://github.com/c-proof/pyglider-example-data

**Comment by jklymak** [comment]
We need a way to serve this and have it updated, probably on release of pyglider....
